.circleci/config.yml

```diff
@@ -79,15 +93,27 @@ workflows:
   version: 2
   build:
     jobs:
+      - "flake8":
+          filters:
+            tags:
+              only: /^v.*$/
```

Comment:
👍 

As I commented in https://github.com/pygae/galgebra/pull/232 , this is really tricky. And this is a little bit annoying, i.e. `tags.only` pollutes the whole way down.